### PR TITLE
added Image attribute in vendor.js in backend

### PR DESCRIPTION
**Pull request by kunaal-gupta**
I have added Image attribute in vendor.js in backend. Let me know if it doesn't work out

### CODE REVIEW

**Review by MoyinF — APPROVED** [review]
Looks good.

**Comment by MoyinF** [comment]
Side note: are Youssef and I code owners here? I should have the option to merge after approving it

**Comment by dimassudjito** [comment]
@MoyinF 
> Side note: are Youssef and I code owners here? I should have the option to merge after approving it

Yes, you and Yuossef are code owners for the entire repo, does it not allow you to merge it?

File: mobile/package.json [hunk 1]
```diff
@@ -17,27 +17,27 @@
...
+    "expo-camera": "~12.3.0",
+    "expo-constants": "^13.2.4",
+    "expo-font": "~10.2.0",
+    "expo-random": "~12.3.0",
+    "expo-splash-screen": "~0.16.2",
+    "expo-status-bar": "~1.4.0",
+    "jest-expo": "^42.1.0",
+    "react": "18.0.0",
+    "react-dom": "18.0.0",
+    "react-native": "0.69.6",
```
Comment (dimassudjito) [inline mobile/package.json:32]: I think updating the whole version could cause trouble for other people with older expo version. I know you're having problem with versioning and I'd suggest downgrading your expo version

File: backend/src/models/Vendor.js [hunk 2]
```diff
@@ -15,6 +15,12 @@ const VendorSchema = new mongoose.Schema(
         phone_number: {
             type: String,
         },
+        point_of_contact: {
+            type: String,
+        },
+        image: {                // Added this
```
Comment (dimassudjito) [inline backend/src/models/Vendor.js:21]: delete unnecessary comment

File: mobile/package.json (hunk 1 above)
Comment (dimassudjito) [reply mobile/package.json:32]: my expo version: 6.0.1

**Review by dimassudjito — CHANGES_REQUESTED** [review]
In general, we want to make the PR as specific as possible so don't include changes unrelated to the ticket. For example, your ticket only include backend work so there's no need to update the mobile folder unless we need to

**Review by dimassudjito — APPROVED** [review]

**Pull request merged**